WcaOnRails/app/models/competition.rb

```diff
@@ -107,7 +108,7 @@ def delegates_must_be_delegates
     end
   end
 
-  def warnings
+  def warnings_for(user)
```

Comment:
I thought a helper file was just some more ruby code. How would it help with the `html_safe` stuff?